charts/lagoon-core/ci/linter-values.yaml

```diff
@@ -15,6 +15,7 @@ s3BAASAccessKeyID: baas
 s3BAASSecretAccessKey: baas123
 lagoonUIURL: http://ui:9101
 lagoonWebhookURL: http://webhook:11213
+defaultIngressClass: nginx
```

Comment:
This doesn't have any effect since no ingress objects are enabled AFAIK.

Reply:
yah, I've left it in, just to ensure that someone looking copies it :wink: